2733.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var findNonMinOrMax = function (nums) {
+  const sortedNums = nums.sort((a, b) => a - b);
+
+  if (sortedNums.length > 2) {
+    return sortedNums[1];
+  } else {
+    return -1;
+  }
```

Comment:
```suggestion
  return sortedNums.length > 2 ? sortedNums[1] : -1;
  }
```
You can also use `Conditional (ternary) operator` so that you can write less code.

Ref: [Conditional operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator)